migrations/2_delploy_ownedresolver_then_ownedregistrar.js

```diff
@@ -0,0 +1,8 @@
+var OwnedResolver = artifacts.require("@ensdomains/resolver/contracts/OwnedResolver.sol");
+var OwnedRegistrar = artifacts.require("./OwnedRegistrar.sol");
+
+module.exports = function(deployer) {
+  deployer.deploy(OwnedResolver).then(function() {
+    return deployer.deploy(OwnedRegistrar, "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e", OwnedResolver.address);
```

Comment:
Is `OwnedResolver` actually used anywhere here?

Reply:
I changed the way the resolver is set up. I think using the public resolver is probably a more reasonable solution. If the TLD holder need to change the resolver in the future, he can do so using the setResolver function.